cmd/pr-reminder/main.go

```diff
@@ -221,18 +221,26 @@ func (p prRequest) link() string {
 }
 
 func (p prRequest) createdUpdatedMessage() string {
+	if time.Since(p.LastUpdated).Hours() <= 24 {
```

Comment:
Let's actually move the emoji to the front of the line so it is more obvious. This will also help to reduce the code duplication here.